manifests/l/Libretro/RetroArch/Nightly/1.10.3/Libretro.RetroArch.Nightly.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22598-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Libretro.RetroArch.Nightly
+PackageVersion: 1.10.3
+PackageLocale: en-US
+Publisher: LibRetro
```

```suggestion
Publisher: Libretro
```
This must match Control Panel.